packages/lib/src/vis/line/LineVis.tsx

```diff
@@ -153,12 +153,14 @@ function LineVis(props: Props) {
                 </div>
 
                 {auxiliaries.map(({ label, array, errors }, index) => (
-                  <div className={styles.tooltipAux} key={label}>
+                  <div className={styles.tooltipAux} key={label || index}>
                     <span
                       className={styles.mark}
                       style={{ color: auxColors[index % auxColors.length] }}
                     />
-                    {label} = {formatTooltipVal(array.get(xi))}
+                    {`${label ? `${label} =` : ''}${formatTooltipVal(
+                      array.get(xi)
+                    )}`}
```

Comment:
You might be able to keep this as two separate JSX expressions (without a space between the two brackets `}{`) 🤷

Reply:
True, it looks ugly here